modules/compiler/utils/source/barrier_regions.cpp

```diff
@@ -971,8 +971,7 @@ void compiler::utils::Barrier::MakeLiveVariableMemType() {
     // Check if the alloca has a debug info source variable attached. If
     // so record this and the matching byte offset into the struct.
 #if LLVM_VERSION_GREATER_EQUAL(18, 0)
-    SmallVector<DbgDeclareInst *, 1> DbgIntrinsics;
-    findDbgDeclares(DbgIntrinsics, member.value);
+    auto DbgIntrinsics = findDbgDeclares(member.value);
```

Comment:
Isn't this the same as >= 17? Can't we just remove the 18 part altogether here since auto resolve it all?

Reply:
LLVM 17 has `auto DbgIntrinsics = FindDbgDeclareUses(member.value);` rather than `auto DbgIntrinsics = FindDbgDeclares(member.value);` (different function name).